scripts/vlan_pool.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+from collections import defaultdict
+from kytos.core.db import mongo_client
+
+DEFAULT_TAG_RANGES = [[1, 4095]]
+
+def get_range(vlans, avoid) -> list[list[int]]:
+    """Convert available_vlans to available_tags.
+    From list[int] to list[list[int]]"""
+    result = []
+    if not vlans:
+        return result
+
+    i = 0
+    while i < len(vlans):
+        if vlans[i] in avoid:
+            i += 1
+        else:
+            break
```

Comment:
`available_vlans` isn't guaranteed to be sorted, and it might not be contiguous either (so it might leave a hole here), I believe just fast forwarding to the right side of non avoided tags it might backfire. If you can double check this.

Reply:
To create `available_tags` correctly, `available_vlans` needs to be sorted. If it is not contiguous it is not a problem, a new ranges is created. For example:
`[2,3,4,20,21,100] -> [[2, 4], [20, 21], [100, 100]]`
Added `vlans.sort()` to guarantee order.

Reply:
The script looks good, a final minor piece of advice is to also add "preview|aggregate" commands just so network operators can double check what's going to be written/updated, this isn't a must have, **it's up to you and how comfortable you are supporting this change**. I'd rather be extra safe when updating production data, and that's why I typically ship preview|aggregate commands, its up to you though. We've used this pattern on a prior topology script before https://github.com/kytos-ng/topology/blob/master/scripts/unset_active.py#L91-L93.

Reply:
Modified to support two different options `aggregate_outdated_interfaces` and `update_database`